polls/models.py

```diff
@@ -14,6 +14,9 @@ class Question(models.Model):
     def was_published_recently(self):
         now = timezone.now()
         return now - datetime.timedelta(days=1) <= self.pub_date <= now
+    was_published_recently.admin_order_field = 'pub_date'
```

Comment:
It was like that in the tutorial. I think it is just an example of how to enable sorting for fields that are not in database so the order shouldn't matter.